### ISSUE TITLE
A couple of the Pokémon blurbs are downright bizarre

### ISSUE THREAD

**Issue by ketsuban** [pixelastic/pokemonorbigdata#18]
(I am apparently one of about three people whose sphere of cognizance includes both Pokémon and Big Data.)

`Feebas requires Swift Swim to get marvel scale.` is nonsensical, because the line is out of context. A Feebas with the ability Swift Swim will evolve into a Milotic with the ability Marvel Scale, which is what the original source (presumably a forum post, given the inconsistent capitalisation and slightly poor English) was trying to say. Since this is hard to make relevant to a light-hearted quiz about Big Data names being incoherent, I'd suggest replacing it with something else; here's a Pokédex entry, either to inspire a joke about Big Data or as a substitute for one.

> Feebas’s fins are ragged and tattered from the start of its life. Because of its shoddy appearance, this Pokémon is largely ignored. It is capable of living in both the sea and in rivers.

`Munchlax is a great counter to Vulpix, being able to take four Fire Blasts from the Life Orb set and hit back hard with Earthquake or STAB Return.` How in the world did a sentence from a Little Cup competitive analysis on the site Smogon.com end up here?! This is clearly ridiculous; here's another Pokédex entry

> Inside Vulpix's body burns a flame that never goes out. During the daytime, when the temperatures rise, this Pokémon releases flames from its mouth to prevent its body from growing too hot.

but if there's a Big Data tool called Lock-On (or similar) you might be interested to know that Vulpix's Japanese name is *Rokon* (ロコン).

**Comment by pixelastic** [comment]
Hello @Ketsuban and thanks for the detailed comment.

As surprising as it may sound, I'm no expert of either big data or pokemon. This whole website was a fun experiment I did a few years ago to learn React (the js library). I used [this google poll](https://docs.google.com/a/algolia.com/forms/d/1kckcq_uv8dk9-W5rIdtqRwCHN4Uh209ELPUjTEZJDxc/viewform?edit_requested=true) as the initial source for questions and added pictures.

The website became popular without me really knowing why and many people left issues about the names of the pokemons not being the correct translations, or suggesting new entries/descriptions.

I'd be happy to add your suggestions/edits to the website as well. As I said, I really don't know enough about either subject to write the entries myself in something that would make sense. I'd be happy to accept pull requests, though.
